conf/prod.routes

```diff
@@ -1,3 +1,6 @@
+# Declare our custom health check first to ensure the deployment fails if migrations fail
+GET     /ping/ping        uk.gov.hmrc.exports.controllers.CustomHealthController.ping()
```

Comment:
Does defining this before the `health.Routes` override the standard one (I would have assumed you had to define it after to override, but I may be wrong) :thinking:

Reply:
First route declared has priority https://www.playframework.com/documentation/2.8.x/ScalaRouting#Routing-priority